models/dodo/optimism/dodo_optimism_schema.yml

```diff
@@ -0,0 +1,199 @@
+      - &token_bought_symbol
+        name: token_bought_symbol
+        description: "Token symbol for token bought in the trade"
+      - &token_sold_symbol
+        name: token_sold_symbol
+        description: "Token symbol for token sold in the trade"
+      - &token_pair
+        name: token_pair
+        description: "Token symbol pair for each token involved in the trade"
+      - &token_bought_amount
+        name: token_bought_amount
+        description: "Value of the token bought at time of execution with its decimals factored in"
+      - &token_sold_amount
+        name: token_sold_amount
+        description: "Value of the token sold at time of execution with its decimals factored in"
+      - &token_bought_amount_raw
+        name: token_bought_amount_raw
+        description: "Raw value of the token bought at time of execution without division by its decimals"
+      - &token_sold_amount_raw
+        name: token_sold_amount_raw
+        description: "Raw value of the token sold at time of execution without division by its decimals"
+      - &amount_usd
+        name: amount_usd
+        description: "USD value of the trade at time of execution"
+      - &token_bought_address
+        name: token_bought_address
+        description: "Contract address of the token bought"
+      - &token_sold_address
+        name: token_sold_address
+        description: "Contract address of the token sold"
+      - &taker
+        name: taker
+        description: "Address of trader who purchased a token"
+      - &maker
+        name: maker
+        description: "Address of trader who sold a token."
+      - &project_contract_address
+        name: project_contract_address
+        description: "Project contract address which executed the trade on the blockchain"
+      - &tx_hash
+        name: tx_hash
+        description: "Unique transaction hash value tied to each transaction on the DEX"
+      - &tx_from
+        name: tx_from
+        description: "Address which initiated the trade"
+      - &tx_to
+        name: tx_to
+        description: "Address which received the trade"
+      - &trace_address
+        name: trace_address
+        description: ""
+      - &evt_index
+        name: evt_index
+        description: "Index of the corresponding trade event"
+
+  - name: dodo_pools_optimism_trades
+    meta:
+      blockchain: optimism
+      sector: dex
+      project: dodo_pools
+      contributors: owen05
+    config:
+      tags: ['optimism', 'dodo', 'pools', 'trades', 'dex']
+    description: >
+        DodoV1 && DODOV2 pools' trades on Optimism.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+      - check_dex_seed:
+          blockchain: optimism
+          project: DODO
+          version: 1
```

Comment:
By the way, which part in the sql is corresponding to DODO version 1?

Reply:
There is no version 1  yet. RD team will add soon